store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -87,20 +89,16 @@
  * @see Graph
  */
 public class FederatedStore extends Store {
-    public static final String USER_IS_ATTEMPTING_TO_OVERWRITE_A_GRAPH_WITHIN_FEDERATED_STORE_GRAPH_ID_S = "User is attempting to overwrite a graph within FederatedStore. GraphId: %s";
     protected static final String S1_WAS_NOT_ABLE_TO_BE_CREATED_WITH_THE_SUPPLIED_PROPERTIES_GRAPH_ID_S2 = "%s was not able to be created with the supplied properties.%n%s";
-    private static final String GAFFER_FEDERATED_STORE = "gaffer.federatedstore.";
-    private static final String GRAPH_IDS = "graphIds";
     private static final String SCHEMA_DEL_REGEX = Pattern.quote(",");
-    private static final String DOT = ".";
-    private static final String FILE = "file";
-    private static final String SCHEMA = "schema";
-    private static final String PROPERTIES = "properties";
-    private static final String ID = "id";
-    private final Map<String, Graph> graphs = Maps.newHashMap();
-    private Set<StoreTrait> traits = new HashSet<>();
+    public static final GraphConfigEnum SCHEMA = GraphConfigEnum.schema;
+    public static final GraphConfigEnum PROPERTIES = GraphConfigEnum.properties;
+    public static final LocationEnum ID = LocationEnum.id;
+    public static final LocationEnum FILE = LocationEnum.file;
+    private FederatedGraphStorage graphStorage = new FederatedGraphStorage();
     private Set<String> customPropertiesAuths;
 
+
```

Comment:
You might need to update this initialise method now you have FederatedStoreProperties.
You could do something like:
```java
if(!(properties instanceof FederatedStoreProperties)) {
    properties = new FederatedStoreProperties(properties);
}
````

Reply:
👍